www/calendar/inner.js

```diff
@@ -907,15 +926,56 @@ define([
                 common.displayAvatar($(avatar), team.avatar, team.displayName || team.name);
                 APP.$calendars.append(h('div.cp-calendar-team', [
                     avatar,
-                    h('span.cp-name', {title: team.name}, team.name)
+                    h('span.cp-name', { title: team.name }, team.name)
                 ]));
-                calendars.forEach(function (id) {
-                    makeCalendarEntry(id, teamId);
-                });
+                if (isMobileView) {
+                    createShowCalendarsButton(teamId, filter, $calendars);
+                } else {
+                    calendars.forEach(function (id) {
+                        makeCalendarEntry(id, 1);
+                    });
+                }
             });
+        }
+        function createShowCalendarsButton(teamId, filter, $parentContainer) {
+            var $showCalendarsContainer = $('<div class="cp-calendar-entry cp-ghost"></div>').appendTo($parentContainer);
```

Comment:
Try to use hyperscript in these cases, even if you need a jQuery object:
`$(h('div.cp-calendar-entry.cp-ghost'))`